src/labels/CustomLabels.labels

```diff
@@ -4865,13 +4945,38 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>Template Info</value>
     </labels>
     <labels>
-        <fullName>geToastTemplateCreateSuccess</fullName>
```

Comment:
@psadie-sfdo If I'm reading this correctly, it looks like it changed to the geTextListViewItemCount custom label. Did I get that right? Might be good to get a little demo of this change. I added a comment in the custom labels spreadsheet too.

Reply:
@melissabarber  - super late response! No change in this label actually. This diff just looks a little wonky. `geToastTemplateCreateSuccess` just got moved down a bit farther to line 4998 because the order changed as I committed additional labels in this case `geTextListViewItemCount`.